pb/edge_ctrl_pb/edge_ctrl.proto

```diff
@@ -33,6 +33,7 @@ message ClientHello {
     string hostname = 2;
     repeated string protocols = 3;
     map<string, string> data = 4;
+    repeated string protoports = 5;
```

Comment:
We can use camel case here and use the full name.

`protocolPorts`

Reply:
I changed 'protoports' to 'protocolPorts' everywhere